extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -505,11 +505,32 @@ export class AmpLightboxViewer extends AMP.BaseElement {
     this.topBar_.appendChild(button);
   }
 
+  /**
+   * Check to see if the triggering click happened on something that is a button
+   * or any other element that should consume the event.
+   * @param {!Event} e
+   * @return {boolean}
+   */
+  shouldTriggerClick_(e) {
+    const target = dev().assertElement(e.target);
+    const consumingElement = closest(target, element => {
```

Comment:
please add `a` and anything that matches `element.getAttribute('on').match(/(^|;)\s*tap\s*:/);` to the predicates.